guide/blueprints/example_yaml/vanilla-bash-netcat-w-client.yaml

```diff
@@ -66,7 +66,7 @@ services:
       name: sayHiNetcat
       description: Echo a small hello string to the netcat entity
       command: |
-        echo $message | nc $TARGET_HOSTNAME 4321
+        echo $message | nc -N $TARGET_HOSTNAME 4321
```

Comment:
What the `-N` does? Also, I'm not sure this is advisable as some platform, i.e. MacOS don't have this option available

Reply:
Struggling to find how you would install an updated version of netcat on MacOS that would support this command. The problem is that on some platforms the `nc` command doesn't close the channel when `echo` terminates, which prevents the pipeline command from terminating.

Would a suitable solution be to include both options with one of them commented out and a comments saying `# MacOS` and `# Linux`?
e.g.
```yaml
# Uncomment the appropriate command for your operating system
# Linux
# echo $message | nc -N $TARGET_HOSTNAME 4321
# MacOS
# echo $message | nc $TARGET_HOSTNAME 4321
```

Reply:
Yeah, or just add a note at the bottom to say that for MacOS, you need to remove `-N` 👍